clients/spark/core/src/main/scala/io/treeverse/clients/ApiClient.scala

```diff
@@ -53,11 +53,19 @@ private object ApiClient {
   }
 }
 
-class ApiClient(apiUrl: String, accessKey: String, secretKey: String) {
+class ApiClient(
+    apiUrl: String,
+    accessKey: String,
+    secretKey: String,
+    connectionTimeout: String = ""
```

Comment:
not sure if necessary to set a default value here

Reply:
There is use of this client in export too, so I added this default value instead of passing none to all places.